amm/src/main/scala/ammonite/interp/Preprocessor.scala

```diff
@@ -293,12 +309,12 @@ object Preprocessor{
 package ${pkgName.map(_.backticked).mkString(".")}
 ${importBlock(imports)}
 
-object ${indexedWrapperName.backticked}{\n"""
+object ${indexedWrapperName.backticked}{\n""".replace("\n", newLine)
```

Comment:
Since this string (actually whole file) is written on a linux machine the formatting due `"""` puts `\n`s after each line of in string. like 
`"""
This line

next line
""" is "\nThisLine\n\nnext line\n"